src/pages/Introduce/Introduce.js

```diff
@@ -0,0 +1,38 @@
+import "./Introduce.scss";
+import { Link, Navigate, UseNavigate, useNavigate } from "react-router-dom";
+
+const Introduce = () => {
+  const navigate = useNavigate();
+
+  const handleInput = () => {
+    console.log("input 태그 동작");
+  };
+
+  const handleButton = () => {
+    console.log("button 태그 동작");
+    navigate("/product");
+  };
+
+  return (
+    <main>
```

Comment:
```suggestion
    <main classNmae="introduce">
```
가장 부모 태그에 className을 컴포넌트의 이름으로 부여해주세요!